### ISSUE TITLE
Remove source JARs from build

### ISSUE THREAD

**Issue by jhpedemonte** [maqetta/maqetta#3524]
Noticed that we have some unneeded JARs in our build.

First, we have quite a few "source" JARs.  Should build without them:

```
org.apache.commons.fileupload.source_1.2.0.M7_0_1.jar
org.eclipse.orion.client.core.source_0.2.0.v20120622-1536.jar
org.eclipse.orion.client.editor.source_0.1.0.v20120618-1815.jar
org.eclipse.orion.client.git.source_0.2.0.v20120620-1957.jar
org.eclipse.orion.client.users.source_0.2.0.v20120605-1950.jar
org.eclipse.orion.doc.isv.source_0.2.0.v20120530-1934.jar
org.eclipse.orion.doc.user.source_0.2.0.v20120413-2008.jar
org.eclipse.orion.server.authentication.basic.source_0.2.0.v20120605-2104.jar
org.eclipse.orion.server.authentication.form.core.source_0.3.0.v20120619-1547.jar
org.eclipse.orion.server.authentication.formopenid.source_0.3.0.v20120619-1547.jar
org.eclipse.orion.server.authentication.openid.core.source_0.3.0.v20120618-1550.jar
org.eclipse.orion.server.configurator.source_0.5.0.v20120612-1830.jar
org.eclipse.orion.server.core.source_0.3.0.v20120619-1547.jar
org.eclipse.orion.server.git.source_0.4.0.v20120611-0901.jar
org.eclipse.orion.server.hosting.source_0.2.0.v20120615-1625.jar
org.eclipse.orion.server.logback.config.source_0.2.0.v20111007-1713.jar
org.eclipse.orion.server.search.source_0.5.0.v20120607-1524.jar
org.eclipse.orion.server.servlets.source_0.5.0.v20120925-1803-bug389754.jar
org.eclipse.orion.server.useradmin.source_0.2.0.v20120210-1503.jar
org.eclipse.orion.server.user.securestorage.source_0.2.0.v20120210-1503.jar
```

Most of these are coming from the **orion build**, so need to change something there.

Also noticed some duplicates:

```
org.cyberneko.html_1.9.14.v201105210654.jar
org.cyberneko.html_1.9.15.jar
```

One comes from **orion**, I believe, and the other from **zazl**.  Can we just make Zazl use the one provided by Orion?

@peller @childsb @JonFerraiolo

**Comment by peller** [comment]
Regarding Orion, perhaps a bug should be filed at bugs.eclipse.org to further reduce the release size, as a companion to the one reported in #3432

Regarding cyberneko, see discussion in #3393

**Comment by jhpedemonte** [comment]
-> M9

**Comment by peller** [comment]
Has a bug been filed with Orion?

**Comment by peller** [comment]
https://bugs.eclipse.org/bugs/show_bug.cgi?id=398262

**Comment by jhpedemonte** [comment]
Fixed by cff4594e6ec99a9b6dd4d60aad975c7640ce0d26.
